cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -160,4 +160,118 @@ TEST(TestGdvFnStubs, TestCastFloat8) {
   ctx.Reset();
 }
 
+    TEST(TestGdvFnStubs, TestToCharFromInt32) {
+        gandiva::ExecutionContext ctx;
+        uint64_t ctx_ptr = reinterpret_cast<int64_t>(&ctx);
+        int32_t out_len = 0;
+
+        const char* out_str = gdv_fn_to_char_int32_int64(ctx_ptr, -46, 100, "%1", &out_len);
```

Comment:
Can you check some other examples for format.  I'm not sure the library that is being used uses the same specification for format strings a SQL to_char would.

Reply:
The implementation is aiming to use the Java formatting.

Reply:
I thought this was mean to mimic oracle's to_char?  if Java formatting is the target, maybe include more examples from that?  I'm not sure the boost formatting library has the same formatting options with the same characters.